src/main/java/frc/robot/commands/AutoIntakeCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CrocodileSubsystem;
+
+public class AutoIntakeCommand extends CommandBase{
+    private final CrocodileSubsystem crocodileSubsystem;
+    private final double speed;
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed){
+        this.crocodileSubsystem = crocodileSubsystem;
+        this.speed = speed;
+        addRequirements(crocodileSubsystem);
+    }
+    @Override
+    public void initialize() {
+        crocodileSubsystem.setIntakeSpeed(speed);
+    }
+    @Override
+    public boolean isFinished() {
+        Debouncer debouncer = new Debouncer(0.5, DebounceType.kBoth);
```

Comment:
This will never return `true` as you are making a new debouncer everytime it calls is finished. You want the debouncer to be persistent beyond the `isFinished` to make sure it can see the value through multiple loops. But im also unsure you need a debouncer here, maybe one beam break trip is good enough.